### ISSUE TITLE
[1.10.2][1.0.3] Custom recipes not working in the lathe

### ISSUE THREAD

**Issue by codetaylor** [zmaster587/AdvancedRocketry#323]
I haven't tested other machines, but I haven't been able to get custom recipes working in the lathe. I want to replace some default recipes and I've tried both:

```
<Recipes defaultRecipes="true">
```

and

```
<Recipes defaultRecipes="false">
```

All of the default recipes still show up in JEI and are still used by the lathe.

Additionally, all of the new recipes that I've added like:

```
<Recipe timeRequired="100" power="40">
  <input>
    <oreDict>plankWood 1</oreDict>
  </input>
  <output>
    <itemStack>minecraft:stick 4 1</itemStack>
  </output>
</Recipe>
```

show up in JEI, but are not used by the machine. Also, in the previous recipe, the `minecraft:stick` texture shows up broken in JEI.

Please let me know if I'm doing something wrong, thanks. :)

**Comment by zmaster587** [comment]
Two things here.  First the default config I supplied was bad which is why changing "defaultRecipes" has no effect.  This better described in #308 .

The stick probably showing up weird because you are trying to use meta value 1 for the stick which doesn't exist in default minecraft to my knowlege.  You can probably just omit the meta value and it'll work like the following:
```
<output>
<itemStack>minecraft:stick 4</itemStack>
</output>
```

See if that works and let me know if they don't.

**Comment by codetaylor** [comment]
Changing `defaultRecipes` to `useDefault` solved that issue.

I can't believe I didn't see that meta value there! Removing the meta value fixed the broken texture.

I tested all of the recipes that I added and all of the oredict ingot to stick/rod recipes work, however, the oredict `plankWood` to `minecraft:stick` recipe does not work. Any planks that I place into the input hatch just sit there and the machine doesn't start. When I change the recipe to use the specific `minecraft:planks` as input, it works as expected.

Thank you for your time!
